src/back/shared/paging.hpp

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include "errors.hpp"
+
+#include <userver/components/component_config.hpp>
+#include <userver/components/component_context.hpp>
+#include <userver/server/handlers/http_handler_json_base.hpp>
+#include <userver/utest/using_namespace_userver.hpp>
+
+namespace svetit::paging {
```

Comment:
Отдельный namespace под это наверное не нужен
```suggestion
namespace svetit {
```

Reply:
Готово.